supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -182,10 +182,10 @@ Admonitions should draw the reader’s attention to certain information. Keep ad
 
 Valid admonition types:
 
-- NOTE
-- IMPORTANT
-- WARNING
-- TIP
+NOTE:: Additional information of which users should be aware, but ignoring it does not impact product usage.
+IMPORTANT:: Advisory information that provides information essential to the completion of a task. Users must not disregard this information.
+WARNING:: Ignoring this information can cause system damage, data loss, or a support-related issue. Explain the problem, cause, and offer a solution that works. If available, offer information to avoid the problem in the future or state where to find more information.
```

Comment:
```suggestion
WARNING:: Ignoring this information causes system damage, data loss, or a support-related issue. Explain the problem, cause, and offer a solution that works. If available, offer information to avoid the problem in the future or state where to find more information.
```

`can` might refer to possibility or permission. Better to remove inaccurate terms.